client/components/App.js

```diff
@@ -8,6 +8,8 @@ import PetTypesList from "./PetTypesList.js"
 
 import { Route, Switch, BrowserRouter, Redirect } from "react-router-dom"
 
+import AdoptablePetType from "./AdoptablePetType.js"
```

Comment:
To organize the imports, you can group together external imports (react-router-dom, jquery, etc) from internal component imports (PetTypesList and this AdoptablePetType)